public/styles/styles-login.css

```diff
@@ -0,0 +1,19 @@
+body {
+    background-color:rgb(234, 234, 234);
+}
+
+.login-main-container {
+    width: 50%;
+    text-align: center;
+    margin: auto;
+    background-color: white;
+    border: 1px solid grey;
+    margin-top: 50px;
+    padding-top: 10px;
+}
+i {
```

Comment:
@lsimpson667 If this styling is specifically for the login page, add another class in front of it so that the style doesn't change all of the i tags on the website. Update this to say something like `.login-icon-container i`